Dockerfile.osd.addon

```diff
@@ -0,0 +1,18 @@
+FROM registry.svc.ci.openshift.org/openshift/release:golang-1.13 AS builder
+
+ENV PKG=/go/src/github.com/integr8ly/integreatly-operator/
+WORKDIR ${PKG}
+
+# compile test binary
+COPY make ./make
+COPY pkg ./pkg
+COPY vendor ./vendor
+COPY test ./test
+COPY Makefile ./
+RUN make test/compile/osd
+
+FROM registry.access.redhat.com/ubi7/ubi-minimal:latest
+
+COPY --from=builder /go/src/github.com/integr8ly/integreatly-operator/integreatly-operator-test-harness.test integreatly-operator-test-harness.test
+
+ENTRYPOINT [ "/integreatly-operator-test-harness.test" ]
```

Comment:
We should add this as an image built by opesnhift CI on each PR, will create a jira about that.

Reply:
@mikenairn ok, once a jira is created, I can follow up on this.